### DISCUSSION TITLE
ESP32 - Firmware V4 beta - Question

### DISCUSSION THREAD

**Discussion by lathemaster** [forkineye/ESPixelStick#561] [Q&A]
I have a project where I would like to use the ESPixelStick V3 with an ESP32 processor and the V4 firmware.

I am tring to figure out what additional hadware I would need to use the additional Pixel Ports.
Hopefully someone smarter than I has adlready figured it out.

Thanks in advance
Mike

**Comment by MartinMueller2003** [comment]
You would use an ESP32 D1 Mini that has the same pinout as the ESP8266 and yes they are available. Then you would need to add an additional buffer/level shifter for the extra channel. V4 has an image for this. FYI: There are also commercially available pre-built platforms that do what you are looking for and have images in the V4 supported platforms set.

**Reply by lathemaster** [reply]
As to the commercially available pre-built platforms, most are currently or mostly out of stock for all the usual 2022 reasons

**Comment by lathemaster** [comment]
Kinda what I thought. I got a couple of ESP32 boards in the D1 Mini format.
I was sorta thinking of a board that went between the ESP32 and the PixelStick and brought
the additional I/O pins out to a ribbon cable. Then design a PCB that has the level shifters, 
screw terminals, etc. Not sure to do power on that aux board or use a power distribution board.

**Reply by MartinMueller2003** [reply]
The QuinLed boards are coming in stock shortly. Also, if you are going to develop a based card then you might as well do the entire base card. FYI: V4 is driving a QuinLed board with 8 ports. That is about where the ESP32 maxes out. Make sure you use an ESP32 processor board with PSRAM. That significantly increases the number of supported pixels. Also make sure your base card has an SD card socket on it so you can keep WiFi lag manageable.

**Reply by lathemaster** [reply]
Thanks Martin - I'll probably try to grab a QuinLED board

**Reply by brmeadows2841** [reply]
I know the Quinled quad has four output terminals and the Q1R output can act as a fifth output.
What is the technique to tap the other three outputs on the Quad?

> The QuinLed boards are coming in stock shortly. Also, if you are going to develop a based card then you might as well do the entire base card. FYI: V4 is driving a QuinLed board with 8 ports. That is about where the ESP32 maxes out. Make sure you use an ESP32 processor board with PSRAM. That significantly increases the number of supported pixels. Also make sure your base card has an SD card socket on it so you can keep WiFi lag manageable.



" FYI: V4 is driving a QuinLed board with 8 ports"

 I know the Quinled quad has four output terminals and the Q1R output can act as a fifth output.
What is the technique to tap the other three outputs on the Quad?
